src/main/java/seedu/ui/Ui.java

```diff
@@ -5,39 +5,39 @@ public class Ui {
+        "=====================================================================\n";
     public static final String WELCOME_MESSAGE = "Hello there, Welcome to\n";
     public static final String INSTRUCTIONS = "\nMyLedger is a University financial manager made for students!\n";
     public static final String HELP_MESSAGE = "If this is your first time here, "
             + "Type <help> to learn more about the commands\n";
-    public static final String HELP_PAGE = "Here are the list of commands and their respective functions. "
+    public static final String HELP_PAGE = "\nHere are the list of commands and their respective functions. "
             + "Please take note of the FORMAT that is stated below\n"
             + HORIZONTAL_LINE +
             "\n1. Add an expenditure to the record\n" +
-            "Format: EXPENDITURE_TYPE d/DATE a/AMOUNT s/DESCRIPTION\n" + 
+            "Format: EXPENDITURE_TYPE d/DATE a/AMOUNT p/DESCRIPTION\n" + 
             "There are 7 EXPENDITURE_TYPE: 'Academic', 'Accommodation', 'Entertainment', " +
             "'Food', 'Transport', 'Tuition', 'Other'\n" +
-            "Example: academic d/2023-02-02 a/25.10 s/NUS\n" +
+            "Example: academic d/2023-02-02 a/25.10 p/NUS\n" +
             HORIZONTAL_LINE +
             "\n2. Add a lend/borrow record\n" +
-            "Format: TYPE d/DATE n/NAME a/AMOUNT b/DEADLINE s/DESCRIPTION\n" +
+            "Format: TYPE d/DATE n/NAME a/AMOUNT b/DEADLINE p/DESCRIPTION\n" +
             "TYPE should be either 'lend' or 'borrow'\n"+
-            "Example: lend d/2023-02-02 n/Akshay Narayan a/25.10 b/2023-04-02 s/CS2113\n" +
+            "Example: lend d/2023-02-02 n/Akshay Narayan a/25.10 b/2023-04-02 p/CS2113\n" +
              HORIZONTAL_LINE +
             "\n3. Delete a specified expenditure from the record\n" +
             "Format: delete INDEX\n" +
             "Example: delete 1 \n" +
             HORIZONTAL_LINE +
             "\n4. Edit an expenditure\n"
-            + "Format: edit INDEX d/DATE a/AMOUNT s/DESCRIPTION\n" +
+            + "Format: edit INDEX d/DATE a/AMOUNT p/DESCRIPTION\n" +
             "Cannot change an expenditure type\n" +
-            "Example: edit 2 d/2023-02-15 a/20.00 s/Eat Food\n" +
+            "Example: edit 2 d/2023-02-15 a/20.00 p/Eat Food\n" +
             HORIZONTAL_LINE +
             "\n4. Edit a Lend/Borrow record\n"
-            + "Format: edit INDEX d/DATE n/NAME a/AMOUNT b/DEADLINE s/DESCRIPTION\n" +
+            + "Format: edit INDEX d/DATE n/NAME a/AMOUNT b/DEADLINE p/DESCRIPTION\n" +
             "Cannot change a lend record to a borrow record\n" +
-            "Example: edit 17 d/2023-02-02 n/Akshay Narayan a/25.10 b/2023-04-02 s/CS2040\n" +
+            "Example: edit 17 d/2023-02-02 n/Akshay Narayan a/25.10 b/2023-04-02 p/CS2040\n" +
             HORIZONTAL_LINE +
             "\n4. List all expenditures in the record\n"
```

Comment:
`list` alone would not work anymore, need to include the currency involved. Should reflect the updated version